cdc/sink/mysql/causality_test.go

```diff
@@ -69,6 +71,7 @@ func TestCausality(t *testing.T) {
 }
 
 func TestGenKeys(t *testing.T) {
+	defer leakutil.VerifyNone(t)
```

Comment:
can u explain why we can use t.Parallel() and VerifyTestMain() in mysql related package?

Reply:
VerifyNone is used to identify which tests have coroutine leaks, ref: https://github.com/pingcap/tiflow/issues/5356#issuecomment-1123213475. The root cause is that mysql Sink was not reliably closed. After the PR was fixed, we were able to open `t.Parallel`